CONTRIBUTING.md

```diff
@@ -86,7 +117,7 @@ Boolean variables and return values should use TRUE and FALSE constans instead o
 ```
 ***
 
-Using colon operator (:) for object property and procs access is deprecated.
+Using colon operator (:) for object property and procs access is generally inadviseable
```

Comment:
Inadvisable. Also it is deprecated I'm not sure why you changed that to "generally inadvisable". You shouldn't be using it at all.

Reply:
Deprecated by byond, are you sure? 

I recently found a perfectly good use for it in unifying behaviour over two distinct types, after a safety check to remove all possibility of runtime errors.

Used properly, it can be a useful tool